packages/sit-onyx/src/components/OnyxBottomBar/OnyxBottomBar.vue

```diff
@@ -0,0 +1,70 @@
+<script lang="ts" setup>
+import { useDensity } from "../../composables/density";
+import type { OnyxBottomBarProps } from "./types";
+
+const props = withDefaults(defineProps<OnyxBottomBarProps>(), {
+  hideTopBorder: false,
+});
+
+defineSlots<{
+  /**
+   * Bottom bar content. This slot is left aligned and should be used by buttons and icon buttons.
+   */
+  left?(): unknown;
+  /**
+   * Bottom bar content. This slot is right aligned and should be used by buttons and icon buttons.
+   */
+  right?(): unknown;
+}>();
+
+const { densityClass } = useDensity(props);
+</script>
+
+<template>
+  <div
+    :class="[
+      'onyx-component',
+      'onyx-bottom-bar',
+      densityClass,
+      { 'onyx-bottom-bar--border': !props.hideTopBorder },
+    ]"
+  >
+    <div class="onyx-bottom-bar__left">
+      <slot name="left"></slot>
+    </div>
+    <div class="onyx-bottom-bar__right">
+      <slot name="right"></slot>
+    </div>
+  </div>
+</template>
+
+<style lang="scss">
+@use "../../styles/mixins/layers.scss";
+
+.onyx-bottom-bar {
+  @include layers.component() {
+    position: fixed;
+    bottom: 0;
+    width: var(--onyx-grid-margin-inline, 100%);
+    max-width: var(--onyx-grid-max-width, 100%);
+    background-color: var(--onyx-color-base-background-blank);
+    display: flex;
+    justify-content: space-between;
+    gap: 0.5rem;
+    padding: var(--onyx-spacing-2xs) var(--onyx-grid-margin);
+    overflow-x: auto;
+
+    &--border {
+      border-top: var(--onyx-1px-in-rem) solid var(--onyx-color-component-border-neutral);
+    }
+
+    &__left,
+    &__right {
+      position: relative;
+      display: flex;
+      width: max-content;
+      gap: 0.5rem;
```

Comment:
```suggestion
    gap: var(--onyx-density-xs);
    padding: var(--onyx-spacing-2xs) var(--onyx-grid-margin);
    overflow-x: auto;

    &--border {
      border-top: var(--onyx-1px-in-rem) solid var(--onyx-color-component-border-neutral);
    }

    &__left,
    &__right {
      position: relative;
      display: flex;
      width: max-content;
      gap: var(--onyx-density-xs);
```

Figma uses `--onyx-density-xs` for the gap :)